app/migrations/Version20150825160125.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
```

Comment:
All right. Anyway, after merging #3326 the dates should be set to be after 08.09.2015